controllers/toolchainstatus/toolchainstatus_controller_test.go

```diff
@@ -1835,10 +1835,9 @@ func cachedToolchainCluster(cl runtimeclient.Client, name string, status corev1.
 		},
 		Client: cl,
 		ClusterStatus: &toolchainv1alpha1.ToolchainClusterStatus{
-			Conditions: []toolchainv1alpha1.ToolchainClusterCondition{{
+			Conditions: []toolchainv1alpha1.Condition{{
 				Type:            toolchainv1alpha1.ToolchainClusterReady,
```

Comment:
As mentioned in the other PR, we could replace the special condition type with the generic one - there is no need to have a special type anymore

Reply:
done